mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -534,6 +534,19 @@ def ROCDL_RawBufferAtomicFAddOp :
   let hasCustomAssemblyFormat = 1;
 }
 
+def ROCDL_GlobalAtomicFAddOp :
+  ROCDL_Op<"global.atomic.fadd">,
+  Arguments<(ins LLVM_Type:$ptr,
```

Comment:
Should we go with ` LLVM_PointerInAddressSpace<1>` here to make it more obvious what's going on?

Reply:
and so you don't have to give the pointer type in the assembly format instead of doing a custom thing